.gitignore

```diff
@@ -0,0 +1,4 @@
+Yogesh/HTML/Assig.1/Que2.html
```

Comment:
Why you added this file

Reply:
it was hidden file so got added unknowingly

Reply:
deleting it